app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -9,6 +9,8 @@
 
   meta_tags["govuk:format"] = content_item_hash[:document_type] if content_item_hash[:document_type]
   meta_tags["govuk:schema-name"] = content_item_hash[:schema_name] if content_item_hash[:schema_name]
+  meta_tags["govuk:publishing-state"] = details_hash[:publishing_state] if details_hash[:publishing_state]
+  meta_tags["govuk:current-language"] = content_item_hash[:locale] if content_item_hash[:locale]
```

Comment:
Locale is already in the HTML, `<html lang="en">`. Can the javascript look for it from there instead of adding extra HTML to the page?